iron-component-page.html

```diff
@@ -317,7 +317,7 @@
            */
           transitive: {
             type: Boolean,
-            value: false
+            value: true
```

Comment:
does this mean that we don't need this property anymore? If that is the case, it seems like it shouldn't break anything if we remove it.